recipes/newrelic/apm/php/debian.yml

```diff
@@ -148,12 +144,47 @@ install:
         - task: send_transaction
         - task: cleanup_temp_files
 
+    create_shared:
+      cmds:
+        - |
+          rm -rf {{.TMP_INSTALL_DIR}} 2>/dev/null
+          mkdir -p {{.TMP_INSTALL_DIR}}
+          echo -e "{{.WHITE}}Temporary directory {{.TMP_INSTALL_DIR}} created{{.NOCOLOR}}"
+
+    user_input:
+      cmds:
+        - |
+          if [[ "{{.NEW_RELIC_ASSUME_YES}}" != "true" ]]; then
+            printf "What is the name of your PHP application? (Double-quote (\") is not a valid character in the name) "
+            read -r APPLICATION_NAME
+
+            printf "Restart the PHP web service after PHP agent installation? Selecting 'y' will automatically restart php-fpm, nginx, or apache depending on the service that is detected. Selecting 'n' will require you to manually restart the process when prompted. (y/n)"
+            read -r APPLICATION_RESTART
+            APPLICATION_RESTART=${APPLICATION_RESTART:-y}
+
+            echo "$APPLICATION_NAME $APPLICATION_RESTART" > "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
+          fi
+
     verify_continue:
       cmds:
         - |
+          if [[ "{{.NEW_RELIC_ASSUME_YES}}" != "true" ]]; then
+            read -r APPLICATION_NAME APPLICATION_RESTART < "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
+          else
+            APPLICATION_NAME="{{.NEW_RELIC_APPLICATION_NAME}}"
+            APPLICATION_RESTART="{{.NEW_RELIC_APPLICATION_RESTART}}"
+
+            # Remove '{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}' and re-create it again with
+            # new values to make them available in next tasks.
+            if [ -f "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}" ]; then
+              rm -f "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}" 2>/dev/null
+            fi
+            echo "$APPLICATION_NAME $APPLICATION_RESTART" > "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
+          fi
+
           echo -e "{{.WHITE}}Confirmation:  The following options were selected.  If you wish to change an option, please select 'n' at the prompt and restart the installation."
-          echo -e "* The following application name was selected: {{.NR_PHP_APPLICATION}}"
-          if [ "{{.NR_PHP_RESTART}}" = "y" ]; then
+          echo -e "* The following application name was selected: $APPLICATION_NAME"
+          if [ "$APPLICATION_RESTART" = "y" ]; then
```

Comment:
You should just be able to ref the input vars like before i.e. `{{.NEW_RELIC_APPLICATION_NAME}}`